### DISCUSSION TITLE
How to view pre-release docs?

### DISCUSSION THREAD

**Discussion by jamesbraza** [astral-sh/ruff#8547] [Q&A] [answered]
The ruff docs site is https://docs.astral.sh/ruff/.  It seems to be specified to the latest release of `ruff`, and you can't change version.

Is there some way to view docs generated from the current `main` branch?

**Comment by charliermarsh** [comment, marked as answer]
@jamesbraza -- Not at present, you'd have to build locally.

**Reply by jamesbraza** [reply]
Thanks for speedy reply!

**Reply by charliermarsh** [reply]
We've considered publishing the docs to a URL on every commit (to enable preview during PR), perhaps as part of that we could also work on publishing preview docs to a stable URL somewhere.

**Reply by zanieb** [reply]
We'd get that "for free" with Netlify, I'm not sure what we want do use for documentation builds going forward.

**Reply by charliermarsh** [reply]
Pages and Vercel offer this too. I'd like to stick with Pages while we can. My next choice would be Vercel since we use it for the Astral site.
